be/src/common/overloaded.h

```diff
@@ -0,0 +1,29 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please use starrocks license header for all new files. This is AST license header